docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -0,0 +1,363 @@
+table_env.execute_sql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and statistics for all columns on partition1 and partition2.
+table_env.execute_sql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and statistics for all columns on all partitions.
+table_env.execute_sql("ANALYZE TABLE Orders COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and statistics for column `amount` on partition1.
+table_env.execute_sql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS FOR COLUMNS(amount)");
+
+# Partition table, collect row count and statistics for `amount` and `product` on partition1 and partition2.
+table_env.execute_sql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS FOR COLUMNS(amount, product)");
+
+# Partition table, collect row count and statistics for column `amount` and `product` on all partitions.
+table_env.execute_sql("ANALYZE TABLE Orders COMPUTE STATISTICS FOR COLUMNS(amount, product)");
+```
+{{< /tab >}}
+{{< tab "SQL CLI" >}}
+```sql
+Flink SQL> CREATE TABLE Store (
+> `id` BIGINT NOT NULl,
+> location VARCHAR(32),
+> owner VARCHAR(32)
+> ) with (
+> ...
+> );
+[INFO] Table has been created.
+
+Flink SQL> CREATE TABLE Orders (
+> `id` BIGINT NOT NULl,
+> product VARCHAR(32),
+> amount INT,
+> `sold_year` BIGINT",
+> `sold_month` BIGINT",
+> ) PARTITIONED BY (`sold_year`, `sold_month`)
+> ) with (
+> ...
+> );
+[INFO] Table has been created.
+
+Flink SQL> ANALYZE TABLE Store COMPUTE STATISTICS;
+    
+Flink SQL> ANALYZE TABLE Store COMPUTE STATISTICS FOR ALL COLUMNS;
+
+Flink SQL> ANALYZE TABLE Store COMPUTE STATISTICS FOR COLUMNS(location);
+    
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Orders COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS FOR ALL COLUMNS;
+
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS FOR ALL COLUMNS;
+    
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS FOR COLUMNS(amount);
+    
+Flink SQL> ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS FOR COLUMNS(amount, product);
+    
+Flink SQL> ANALYZE TABLE Orders COMPUTE STATISTICS FOR COLUMNS(amount, product);
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+
+## Syntax
+
+```sql
+ANALYZE TABLE [catalog_name.][db_name.]table_name [PARTITION(partcol1=val1 [, partcol2=val2, ...])] COMPUTE STATISTICS [FOR COLUMNS col1 [, col2, ...] | FOR ALL COLUMNS]
```

Comment:
the syntax should be updated as: `PARTITION (partcol1[=val1] [, partcol2[=val2], ...])` see the FLIP doc

Reply:
> the syntax should be updated as: `PARTITION (partcol1[=val1] [, partcol2[=val2], ...])` see the FLIP doc
Done!